docs/reference/primitives/app/timers.md

```diff
@@ -0,0 +1,69 @@
+# Timer
+
+A Timer spawns a [Process](process.md) on a schedule. The schedule is defined using `cron` syntax. A timer is the equivalent of issuing a [convox run](../../cli/run.md) command but on a defined schedule. An application can have multiple timers defined.
+
+## Definition
+
+A Timer is defined in [`convox.yml`](../../../configuration/convox.yml.md).
+
+    timers:
+      cleanup:
+        schedule: "0 3 * * ? *"
+        command: bin/cleanup
+        service: worker
+
+### Attributes
+
+| Name       | Required | Description                                                           |
+| ---------- | -------- | --------------------------------------------------------------------- |
+| `schedule` | **yes**  | A cron formatted schedule for spawning the process. All times are UTC |
+| `command`  | **yes**  | The command to run that spawns the process                            |
```

Comment:
Process should be capitalized and linked

Reply:
This is a bit awkwardly worded. This command doesn't spawn the Process, maybe the command that runs inside the Process?